charts/zitadel/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+Thank you for installing {{ .Chart.Name }}.
+
+{{- if and (not (empty .Values.zitadel.configmapConfig.FirstInstance)) (and (not (empty .Values.zitadel.configmapConfig.FirstInstance.Org)) (not (empty .Values.zitadel.configmapConfig.FirstInstance.Org.Machine))) }}
```

Comment:
does the following not work?
{{- if and .Values.zitadel.configmapConfig.FirstInstance .Values.zitadel.configmapConfig.FirstInstance.Org .Values.zitadel.configmapConfig.FirstInstance.Org.Machine }}

Reply:
?

Reply:
changed it in all ifs

Reply:
doesn't look changed to me :joy: 
![image](https://user-images.githubusercontent.com/12727842/208081608-3ff1f4dc-11d5-4d79-aff6-debd14b5c068.png)

Reply:
ah you're right, i just thought that the second 'and' was to much, but the 'not' and 'empty' can be removed too